### PR TITLE
UCT/UD: Add UD endpoint information to VFS

### PR DESCRIPTION
**Pull request by yosefe**
## Why ?
Inspect UD endpoint TX/RX state

### CODE REVIEW

File: src/uct/ib/ud/base/ud_iface.c [hunk 1]
```diff
@@ -1063,6 +1065,10 @@ void uct_ud_iface_vfs_refresh(uct_iface_h iface)
     ucs_vfs_obj_add_ro_file(ud_iface, ucs_vfs_show_primitive,
                             &ud_iface->config.tx_qp_len, UCS_VFS_TYPE_INT,
                             "tx_qp_len");
+
+    ucs_ptr_array_for_each(ep, i, &ud_iface->eps) {
+        uct_ud_ep_vfs_populate(ep);
```
Comment (rakhmets) [inline src/uct/ib/ud/base/ud_iface.c:1070]: Please add `ucs_vfs_obj_remove` for each UD endpoint when it is destroyed.

Comment (yosefe) [reply src/uct/ib/ud/base/ud_iface.c:1070]: i think it's already done here:
https://github.com/yosefe/ucx/blob/8d014c0c7a0b6d2fdbddaa25e697828d4d02058c/src/uct/base/uct_iface.c#L607

Comment (rakhmets) [reply src/uct/ib/ud/base/ud_iface.c:1070]: Yes, I see.

**Review by rakhmets — DISMISSED** [review]

File: src/uct/ib/ud/base/ud_ep.c [hunk 2]
```diff
@@ -1800,3 +1802,45 @@ ucs_status_t uct_ud_ep_invalidate(uct_ep_h tl_ep, unsigned flags)
...
+                            UCS_VFS_TYPE_U16 | UCS_VFS_TYPE_FLAG_HEX, "flags");
+
+    ucs_vfs_obj_add_ro_file(ep, ucs_vfs_show_primitive, &ep->tx.psn,
+                            UCS_VFS_TYPE_U16, "tx/psn");
+    ucs_vfs_obj_add_ro_file(ep, ucs_vfs_show_primitive, &ep->tx.max_psn,
+                            UCS_VFS_TYPE_U16, "tx/max_psn");
+    ucs_vfs_obj_add_ro_file(ep, ucs_vfs_show_primitive, &ep->tx.acked_psn,
+                            UCS_VFS_TYPE_U16, "tx/acked_psn");
+    ucs_vfs_obj_add_ro_file(ep, ucs_vfs_show_primitive, &ep->tx.pending.ops,
+                            UCS_VFS_TYPE_U32_HEX, "pending_ops");
```
Comment (rakhmets) [inline src/uct/ib/ud/base/ud_ep.c:1829]: `tx/pending_ops` ?

Comment (yosefe) [reply src/uct/ib/ud/base/ud_ep.c]: fixed

**Review by rakhmets — DISMISSED** [review]

**Pull request merged**